src/constants/relationships.js

```diff
@@ -0,0 +1,8 @@
+const RELATIONSHIPS = {
+  Blank: '',
```

Comment:
A bit confused about the choice of casing? It would seem `src/contants/relationship_status.js` utilizes upper casing for its keys, lower casing for its values, but `src/constants/relationship.js` utilizes capitalization throughout. However, I think it's probably due to the old INTERACTION_TYPES from `src/constants/interaction_types.js` that's causing the confusion. We'll probably need to remove the older `src/constants/interaction_types.js` since it's no longer in use.

After looking over all our `constants` files, I think we should stick with the convention set by `src/contants/HMIS_ONTOLOGY_VALUES.js` and `src/constants/rdf_predicates.js` with camelCasing keys at the very least.